SU-HSDO/mrc_blt
# Jk sandbox

# READY FOR REVIEW

# Summary
- Ensure that the latest of all of the new front-end work is on the main development site for MRC

# Needed By (Date)
- End of sprint (11/22)

# Urgency
- N/A

# Steps to Test

1. After https://github.com/SU-HSDO/stanford_mrc/pull/5 is merged...
1. Pull this branch
2. Verify that the new styles (e.g., bright read global header and cardinal global footer) are present
3. Ensure that Decanter v2.0 is the version of Decanter

# Affected Projects or Products
- MRC
- mrc_blt
- Decanter
- stanford_basic

# Associated Issues and/or People

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
